.editorconfig

```diff
@@ -0,0 +1,11 @@
+# http://editorconfig.org
```

Comment:
Missing changes to README.md.
* Need to show how to configure the library
* How to use the library
* Potentially a sample ServiceAuthTokenFactory implementation